### ISSUE TITLE
How to prevent nested li tags being tied to parent UL

### ISSUE THREAD

**Issue by ollie007** [stazna01/jQuery-at-Accordion-or-Tabs#3]
```
<ul>
<li></li>
<li>

<ul>
<li></li>
<li></li>
</ul>
```

</li>
</ul>

```
</li>
</ul>
```

How to prevent nested li tags being tied to parent UL
We just need the outer UL to be tagged, the others should not be...the nested links are just rotating the tabs at the moment

**Comment by ollie007** [comment]
can someone add the class for the 
    <li class="atAccordionOrTabs">

so that only those list items are activated please

**Comment by stazna01** [comment]
I believe this was addressed in version 1.0.3. I ran into the same issue in a project and everything seemed good after my 1.0.3 changes.

**Comment by ollie007** [comment]
when did you make the change?
